lib/run_by_unit.sh

```diff
@@ -212,12 +222,34 @@ Tstart
 $libDir/split_flows.py $outputHucDataDir/demDerived_reaches.shp $outputHucDataDir/dem_thalwegCond.tif $outputHucDataDir/demDerived_reaches_split.gpkg $outputHucDataDir/demDerived_reaches_split_points.gpkg $maxSplitDistance_meters $slope_min $outputHucDataDir/wbd8_clp.gpkg  $outputHucDataDir/nwm_lakes_proj_subset.gpkg
 Tcount
 
+if [ "$extent" = "MS" ]; then
+  ## MASK RASTERS BY MS BUFFER ##
+  echo -e $startDiv"Mask Rasters with Stream Buffer $hucNumber"$stopDiv
+  date -u
+  Tstart
+  $libDir/fr_to_ms_raster_mask.py $outputHucDataDir/demDerived_reaches_split.gpkg $outputHucDataDir/flowdir_d8_burned_filled.tif $outputHucDataDir/dem_thalwegCond.tif $outputHucDataDir/slopes_d8_dem_meters.tif $outputHucDataDir/flowdir_d8_MS.tif $outputHucDataDir/dem_thalwegCond_MS.tif $outputHucDataDir/slopes_d8_dem_metersMS.tif $ms_buffer_dist
+  Tcount
+
+  if [[ ! -f $outputHucDataDir/dem_thalwegCond_MS.tif ]] ; then
+    echo "No AHPs point(s) within HUC $hucNumber boundaries. Aborting run_by_unit.sh"
+    exit
+  fi
+
+  dem_thalwegCond=$outputHucDataDir/dem_thalwegCond_MS.tif
+  slopes_d8_dem_meters=$outputHucDataDir/slopes_d8_dem_metersMS.tif
```

Comment:
It seems that convention is to add "_MS". Please consider changing slopes_d8_dem_metersMS to slopes_d8_dem_meters_MS (here and other places if necessary) or specify reasoning for keeping.